wdl/transforms/new-base/src/main/scala/wdl/transforms/base/wdlom2wdl/WdlWriterImpl.scala

```diff
@@ -449,12 +449,17 @@ object WdlWriterImpl {
   }
 
   implicit val importElementWriter: WdlWriter[ImportElement] = new WdlWriter[ImportElement] {
-    override def toWdlV1(a: ImportElement): String = "import \"" + a.importUrl + "\""
+    override def toWdlV1(a: ImportElement): String = {
+      a.namespace match {
+        case Some(namespace) => s"""import "${a.importUrl}" as $namespace"""
+        case None =>            s"""import "${a.importUrl}""""
+      }
+    }
   }
 
   implicit val fileElementWriter: WdlWriter[FileElement] = new WdlWriter[FileElement] {
     override def toWdlV1(a: FileElement) = {
-      "version 1.0" +
+      "version 1.0" + System.lineSeparator +
```

Comment:
ToL: Not new in this PR but we should change this to put `workflow`s above `task`s in the upgrade results.

I've been trying to push people towards doing that as a "best practice" so our upgrade script doing the opposite would be a little bit sad!